getting-started/securing-timescale-cloud.md

```diff
@@ -0,0 +1,68 @@
+# How to secure network access to your Timescale Cloud service
+
+It's the easiest question to ask, but the hardest to answer about any system: 
+**is it secure?** Having a secure system doesn't have a binary "yes or no" answer 
+like the question implies. Security is implemented in layers and each choice 
+comes with tradeoffs. And in a world where you are using cloud services with your 
+system, this topic becomes even more important and complex.
+
+That said, one very common and critical piece of any security story is the network. 
+First you must protect your system from the outside and work inward. The most basic 
+network protection that can be applied is to limit the set of ports for access, 
+thus reducing the attack surface area.
+
+Exposing a minimal set of ports is a good start, but another layer you can apply 
+is to protect the inbound traffic to a minimal set of source IP ranges. 
+[Timescale Cloud][timescale-cloud] provides the ability to configure which source 
+IP addresses are allowed to connect your TimescaleDB instance through the connection port.
+
+This tutorial will walk you through how to configure this capability.
+
+### Before you start
+
+Be sure to follow the instructions to [setup Timescale Cloud][timescale-cloud-install] to
+get signed up and create your first database instance.
+
+### Step 1 - Navigate to your TimescaleDB instance
+
+Once you have a database instance setup in the [Timescale Cloud portal][timescale-portal],
+browse to this service and click on the “Overview” tab. In the “Connection Information” 
+section, you will see the port number that is used for database connections. This is 
+the port we will protect by managing inbound access.
+
+<img class="main-content__illustration" src="https://s3.amazonaws.com/docs.timescale.com/securing-timescale-cloud/overview-tab.png" alt="Timescale Cloud Oerview tab"/>
```

Comment:
```suggestion
<img class="main-content__illustration" src="https://s3.amazonaws.com/docs.timescale.com/securing-timescale-cloud/overview-tab.png" alt="Timescale Cloud Overview tab"/>
```

Reply:
I assume this image will be put in at the src location soon?  It's not there now.

Reply:
Screenshots are in Dropbox: https://www.dropbox.com/sh/1s5039hht5qumxx/AAB5uGFgVGjqjgbJRZJsGwZaa?dl=0